DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/staticIndexes/storeFieldInIndex/metadata.json

```diff
@@ -0,0 +1,241 @@
+                },
+                {
+                    "Title": "Studio: Index Field Options",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/java/studio/database/indexes/create-map-index#index-field-options"
+                }
+            ]
+        },
+        {
+            "Title": "Query for documents",
+            "Slug" : "step-6",
+            "DescriptionLines": [
+                "* Query the 'Orders' collection using the defined index.",
+                "",
+                "* The orders' documents can be _filtered_ by the index-fields defined in `OrdersQuantity_ByCompany.IndexEntry`",
+                "",
+                "* The resulting documents will include only orders that where made by 'companyId'."
+            ],
+            "Assets": [
+
+                {
+                    "Title": "Filtering Results - Basics Demo",
+                    "Type" : "demo",
+                    "Url"  : "/java/queries/filtering-results-basics"
+                }
+            ]
+        },
+        {
+            "Title": "Project Results",
+            "Slug" : "step-7",
+            "DescriptionLines": [
+                "* Use `selectFields()` to make the server return only the fields defined in the 'OrdersQuantity_ByCompany.OrderProjectedDetails' class per document result instead of the whole document.<br>",
+                "  * If the field is _stored in the index_ - the server will fetch it from the index<br>",
+                "  * If the field is _not stored in the index_ - the server will fetch it from the document store",
+                "",
+                "* In this example:<br>",
+                "  * 'TotalItemsOrdered' is fetched from the index<br>",
+                "  *  'OrderedAt' if fetched from the document store"
+            ],
+            "Assets": [
+                {
+                    "Title": "Project Index Result Demo",
+                    "Type" : "demo",
+                    "Url"  : "/static-indexes/project-index-results"
+                },
+                {
+                    "Title": "selectFields()",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/indexes/querying/projections#select"
+                }
+            ]
+        },
+        {
+            "Title": "Execute the Query",
+            "Slug" : "step-8",
+            "DescriptionLines": [
+                "* The _ordersQuery_ variable only stores the query definition.<br>",
+                "  Execute the query by calling `toList()`, or any other method that forces immediate execution.",
+                "",
+                "* The final query results include the projected details for orders that where made by 'companyId'.",
+                "",
+                "* The resulting objects are Not tracked by the session since they are not the stored documents but only partial data."
```

Comment:
1. refactor:
the stored documents => the full documents

2. pls fix that for the **C#** demo as well - thanks

Reply:
Done